avatar.css

```diff
@@ -0,0 +1,118 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```

Comment:
You can put this in reset.css so no need to write it again for each component.

Reply:
Yes, I thought about it, I will do this. Thank you